go.mod

```diff
@@ -21,7 +21,9 @@ require (
 	k8s.io/client-go v0.30.2
 	k8s.io/component-base v0.30.2
 	k8s.io/klog/v2 v2.130.0
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.0
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.29.0
```

Comment:
pleaser restore v0.0.0 to avoid confusion. Note there is a replace directive below.

Reply:
Done!